modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/frontend/taglib/servlet/taglib/UserMembershipsScreenNavigationEntry.java

```diff
@@ -48,6 +55,28 @@ public String getJspPath() {
 		return "/user/memberships.jsp";
 	}
 
+	public boolean isEditable(
+		HttpServletRequest httpServletRequest,
+		HttpServletResponse httpServletResponse) {
+
+		ServiceContext serviceContext =
+			ServiceContextThreadLocal.getServiceContext();
+
+		ThemeDisplay themeDisplay = serviceContext.getThemeDisplay();
+
+		PortletDisplay portletDisplay = themeDisplay.getPortletDisplay();
+
+		String myAccountPortletId = PortletProviderUtil.getPortletId(
+			PortalMyAccountApplicationType.MyAccount.CLASS_NAME,
+			PortletProvider.Action.VIEW);
+
+		if (myAccountPortletId.equals(portletDisplay.getPortletName())) {
```

Comment:
Is it correct that we are comparing a portletId with the portletName?

Reply:
Yes, for whatever reason PortletProviderUtil.getPortletId retrieves the portletName. See: https://github.com/liferay-commerce/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/portlet/PortletProviderUtil.java#L38-L40